keyhive_core/src/keyhive.rs

```diff
@@ -1729,12 +1717,31 @@ impl<
         &self,
         events: HashMap<Digest<Event<S, T, L>>, Event<S, T, L>>,
     ) -> Result<(), ReceiveStaticEventError<S, T, L>> {
-        tracing::debug!("executing Keyhive::ingest_event_table()");
         self.ingest_unsorted_static_events(
             events.values().cloned().map(Into::into).collect::<Vec<_>>(),
         )
         .await
     }
+
+    pub async fn stats(&self) -> Stats {
+        let active_prekey_count = self
+            .active
+            .lock()
+            .await
+            .individual
+            .lock()
+            .await
+            .prekey_ops()
+            .len() as u64;
+        Stats {
```

Comment:
No one has complained yet, but I don't love the nested lock situation that I ended up with when making this package sendable ☹️ Nothing to do, just noting it as something that if we find a solution to would be nice to find a workaround.